README.md

```diff
@@ -125,6 +129,62 @@ Kafka. This value cannot be set, only read.
 
 Aiven ID format when importing existing resource: name of the project as is.
 
+### Resource Account
+```
+resource "aiven_account" "foo" {
+    name = "<ACCOUNT_NAME>"
+}
+
+```
+
+`name` defines an account name.
+
+`account_id` is an auto-generated unique account id.
+
+### Resource Account Team
+```
+resource "aiven_account_team" "foo" {
+    account_id = aiven_account.<ACCOUNT_RESOURCE>.account_id
+    name = "account_team1"
+    
+    # invite users to a team
+    invite_users = "user+1@example.com,user+2@example.com"
```

Comment:
This differs from how invitations work for projects. Unless there's some technical reason why team invites must work differently I'd use the same model for both.

This list of invite users seems potentially problematic. What happens when user actually accepts the invite? Then the user isn't invited anymore so Terraform state does not match actual state. Does removing an invite mean that you're removing the team member if the members has already accepted the invite? Having invites handled via team member resource that just has computed accepted property seems better.

Reply:
Yes, indeed, it is different from project invitations and might not be straightforward for a user. 

First of all, I missed that we have a `Delete` method for invitations in Aiven API
```
"/account/<account_id>/team/<team_id>/invites/<user_email>",
method="DELETE"
```

And now it seems that the best idea would be to create a separate TF resource for Account Tema Invitations `account_team_invitation`, and when a user accepts an invitation, we will also check the member's list to make sure invitation was accepted. Perhaps even add a computed field `status` (sent, accepted) which will be calculated based on two records: invitations and members list. 

And `aiven_account_team.member` field will stay as the computed list of members. 

Or do you have something else in mind?

Reply:
Should be `account_team_member` instead of `account_team_invitation`. The `aiven_account_team.member` that I would consider not adding initially and if there's demand add it later. We don't have that for project and I don't think anyone has ever asked for it.

Reply:
Added `aiven_account_team_member` resource and deleted `aiven_account_team.member` and `aiven_account_team.invite`.